packages/components/src/core/InputSearch/index.tsx

```diff
@@ -50,6 +52,19 @@ const InputSearch = forwardRef<HTMLDivElement, InputSearchProps>(
       if (onChange) onChange(event);
     };
 
+    const clearInput = () => {
+      setValue("");
+      setHasValue(false);
+
+      if (handleSubmit) handleSubmit("");
+
+      if (onChange) {
```

Comment:
Same as the Autocomplete component, Because we are manually firing this event, we must build a onChange event to transmit the updated value to onChange listeners.